ts/compile/runner/src/text.ts

```diff
@@ -73,7 +73,7 @@ export class TextDecoder {
     if (a instanceof Uint8Array) {
       return this.decodeUint8Array(a);
     } else {
-      console.log("TextDecoder called with " + typeof(a), a)
+      console.log("TODO: TextDecoder called with " + typeof(a), a)
```

Comment:
Why is this TODO?

Reply:
We do not currently support the entire interface of TextDecoder. We only implement what is required for polyglot to work (Which is the majority of usecases anyway).
But, if someone writes typescript relying on the built in TextDecoder and expects it to work when called with other types, it will trigger this.